webpack.config.example.js

```diff
@@ -32,7 +32,8 @@ const path = require('path');
  * Mapbox GL JS uses the webworkify module by default, and that module
  * does things very specific to browserify's module loading system. In this
  * configuration, we alias webworkify to webworkify-webpack to add webpack
- * support.
+ * support. NOTE: there is a known caveat when using 'eval'-related devtool
+ * settings, make sure to double-check your configuration.
```

Comment:
Can you provide more context here? What is the caveat? What should developers check their configurations _for_?